### PR TITLE
Feature: popover the account overview when hovering on the avatar

### PR DESCRIPTION
**Pull request by thai-d-v**
### Description

I make a popover that will show up when hovering. It provides an overview of that account without going to the personal page.

- using native `popover` API
- support Mac Catalyst and iPad
- the layout is based on `AccountDetailHeaderView`
- follow existing theme adaptive design
- follow existing accessibility convention 

### Demonstration

> You should wait a little bit for the GIF to be loaded.

#### Mac Catalyst

Demo-Account-Overview-Popup--Nov-20-2023 08-41-59

#### iPad

Demo-Account-Overview-Popup-iPad--Nov-20-2023 11-54-59

### CODE REVIEW

File: Packages/DesignSystem/Sources/DesignSystem/Views/AvatarView.swift [hunk 1]
```diff
@@ -2,104 +2,315 @@ import Nuke
...
+
+  public var body: some View {
+    if let account = account {
+      if hasPopup {
+        AvatarImage(account: account, config: adaptiveConfig)
+          .onHover { hovering in
+            if hovering {
+              toggleTask.cancel()
+              toggleTask = Task {
+                try? await Task.sleep(nanoseconds: NSEC_PER_SEC / 2)
```
Comment (Dimillian) [inline Packages/DesignSystem/Sources/DesignSystem/Views/AvatarView.swift:27]: Too bad we don't have a proper SwiftUI way to delay the hover.

**Comment by Dimillian** [comment]
I like it a lot! But I would disable it for the sidebar account avatar view. It's conflicting a bit when you want to switch accounts. Once it's done I'll merge it!

File: Packages/DesignSystem/Sources/DesignSystem/Views/AvatarView.swift (hunk 1 above)
Comment (thai-d-v) [reply Packages/DesignSystem/Sources/DesignSystem/Views/AvatarView.swift:27]: Luckily, We have Swift concurrency. 😁

**Comment by thai-d-v** [comment]
Sorry, that's my bad. I fixed it. 😁

**Pull request merged**